Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM quay.io/openshiftio/fabric8-analytics-f8a-worker-base:aa8893d
 ENV LANG=en_US.UTF-8 \
     # place where to download & unpack artifacts
     WORKER_DATA_DIR='/var/lib/f8a_worker/worker_data' \
+    # place for npm artifacts
+    NPM_DATA_DIR='/workdir/.npm' \
```

Comment:
You may not need to have a separate env for this. By default `npm` stores it's cache in $HOME/.npm directory. In this case `HOME` points to `/workdir`. So, `shutil.rmtree(path.join(os.environ['HOME'], '.npm'))` should be fine.

Reply:
we dont specifically add anything in the .npm folder. its downloading the artifacts implicitly when the npm -registry and other commands are executed during the analysis phase

Reply:
Right. Instead of having a separate env variable, you can derive the path of `.npm` directory based on `HOME` env variable.

Reply:
@arajkumar PTAL at the changed files